Vagrantfile

```diff
@@ -0,0 +1,11 @@
+Vagrant.configure(2) do |config|
+
+    config.vm.box = "ubuntu/trusty64"
+
+    config.vm.provision :shell, :path => "vagrant/bootstrap.sh"
+
+    config.vm.provider "virtualbox" do |v|
+        v.customize ["modifyvm", :id, "--memory", 1024]
+    end
+
```

Comment:
Yep, I meant that. I haven't used Vagrant in a long time so I didn't know that the local directory was already shared.

Thanks :)